docs/robots/outdoor_robots/husky/tutorials_husky.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Husky Tutorials
+sidebar_label: Tutorials
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+import ComponentIntroductionHusky from "/components/introduction_husky.mdx";
+import ComponentTutorialLinks from "../../../components/_tutorials_link.mdx";
+import Support from "/components/support.mdx";
+
+<ComponentIntroductionHusky />
+
+## Husky Overview
+
+### Introduction
+
+Husky is a rugged, outdoor-ready unmanned ground vehicle (UGV), suitable for research and rapid prototyping applications.
+These tutorials will assist you with setting up and operating your Husky.
+
+For more information or to receive a quote, please [visit us online](http://clearpathrobotics.com/husky).
+
+<Highlight color="#25c2a0">Docusaurus green</Highlight> and <Highlight color="#1877F2">Facebook blue</Highlight> are my favorite colors.
+
+
+<ComponentTutorialLinks />
+
+## Jetson Installation
+
+Refer to the [Jetson](../../accessories/computers/jetson/jetson_hardware) page for details on installing a Jetson in Husky.
+However, at present, ROS2 software is not supported on Jetson in Husky.
```

Comment:
Change "ROS2" to "ROS 2" throughout all files

Reply:
Done